app/config/cosmos.js

```diff
@@ -23,7 +30,10 @@ const config = {
   paymentsDatabase: process.env.COSMOS_PAYMENTS_DATABASE,
   paymentsContainer: process.env.COSMOS_PAYMENTS_CONTAINER,
   preferencesDatabase: process.env.COSMOS_PREFERENCES_DATABASE,
-  preferencesContainer: process.env.COSMOS_PREFERENCES_CONTAINER
+  preferencesContainer: process.env.COSMOS_PREFERENCES_CONTAINER,
+  queriesDatabase: process.env.COSMOS_QUERIES_DATABASE,
+  queriesContainer: process.env.COSMOS_QUERIES_CONTAINER,
+  responsesContainer: process.env.COSMOS_RESPONSES_CONTAINER
```

Comment:
Do we need these to reference these in config if they have a default value above? If we do should if be an empty string rather than the env file as they wont exist in the env file anymore.

One to check with @johnwatson484.

Reply:
Yeah I was wondering this too, the PRs on the receivers are now set up so that you no longer need the values in your `.env` so I do think these could be removed like you said.

Reply:
Adding on this point, the same is also the case for the receivers and the query service and they're all currently failing to deploy cause of issues with Cosmos config.

Reply:
There is no cosmos in the SND environment to connect to so would expect it them to fail at moment

Reply:
For those we need to add some logic to check cosmos is available and if not dont want to attempt to connect but we can pick that up separately

Reply:
So to resolve this, you can remove tall references to databases and containers from the config object (lines 26-36) but we need to leave them in the schema

Reply:
Done :)